### DISCUSSION TITLE
Is llamaparse available for on premise deployment

### DISCUSSION THREAD

**Discussion by pulkitmehtaworkmetacube** [run-llama/llama_cloud_services#369] [General]
Is llamaparse available for on premise deployment since we are working in LLM Data Security realm and cannot use managed API .

**Comment by MatthewEvolving** [comment]
Same question, thanks!

**Comment by apostolos-geyer** [comment]
@MatthewEvolving  @pulkitmehtaworkmetacube 
AFAIK LlamaParse can be deployed on-prem as part of the broader suite of LlamaCloud tooling. The [LlamaCloud docs for self hosting](https://docs.cloud.llamaindex.ai/llamacloud/self_hosting/architecture) indicate this is an enterprise only feature.
